PanCardViewSample/PanCardViewSample/Views/CardsSampleView.cs

```diff
@@ -20,15 +20,15 @@ public CardsSampleView()
 			var prevItem = new ToolbarItem
 			{
 				Text = "**Prev**",
-				IconImageSource = "prev",
+				IconImageSource = Device.RuntimePlatform != Device.Tizen ? "prev" : "prev.png",
 				CommandParameter = false
 			};
 			prevItem.SetBinding(MenuItem.CommandProperty, nameof(CardsSampleViewModel.PanPositionChangedCommand));
 
 			var nextItem = new ToolbarItem
 			{
 				Text = "**Next**",
-				IconImageSource = "next",
+				IconImageSource = Device.RuntimePlatform != Device.Tizen ? "next" : "next.png",
```

Comment:
```suggestion
				IconImageSource = "next.png",
```
I guess, we can unify image source naming. By this way, it should work for all platforms